src/tools/perf/perftest_params.c

```diff
@@ -73,6 +73,7 @@ static void usage(const struct perftest_context *ctx, const char *program)
     printf("     -O <count>     maximal number of uncompleted outstanding sends\n");
     printf("     -i <offset>    distance between consecutive scatter-gather entries (%zu)\n",
                                 ctx->params.super.iov_stride);
+    printf("     -l             use loopback connection\n");
```

Comment:
pls elaborate what it means
also, does it work that you can run ucx_perftest without specifying server's IP address?

Reply:
yes, it works

Reply:
if (rte_group_rank(group) == rte_group_size(group) - 1)
Don't think it's a good idea\, let's make a readable code - ctx->params.super.flags & UCX_PERF_TEST_FLAG_LOOPBACK is more clear to understand

Reply:
in additional - two calls for rte functions vs. one bit operation

Reply:
I think that the option to specify server IP address together with `-l` is confusing (for the user).
Maybe allow only one of them? - And reflect that in the usage string, so that it is clear that `-l` can be used instead of the server hostname.

Reply:
agree with @gleon99  - should allow either '-l' , or a server address (mutually exclusive) - and document this in the "usage" printout.